### PR TITLE
labels: fix lib/src bucket for src/ changes

### PR DESCRIPTION
**Pull request by phillipj**
The `lib / src` label bucket / limit does currently not work on files in `src/`.

Closes https://github.com/nodejs/github-bot/issues/137

### CODE REVIEW

File: test/unit/node-labels.test.js [hunk 1]
```diff
@@ -235,6 +235,39 @@ tap.test('label: "lib / src" when 5 or more JS sub-systems have been changed', (
...
+    'src/node_serdes.cc',
+    'src/node_zlib.cc',
+    'src/process_wrap.cc',
+    'src/signal_wrap.cc',
+    'src/string_bytes.cc',
+    'src/timer_wrap.cc',
+    'src/uv.cc'
+  ])
+
+  t.same(labels, ['c++', 'lib / src'])
```
Comment (phillipj) [inline test/unit/node-labels.test.js]: @Fishrock123 @mscdex @addaleax does this look reasonable?

Tried mimicking what was manually set on https://github.com/nodejs/node/pull/12366..

File: test/unit/node-labels.test.js [hunk 2]
```diff
@@ -235,6 +235,39 @@ tap.test('label: "lib / src" when 5 or more JS sub-systems have been changed', (
   t.end()
 })
 
+// https://github.com/nodejs/node/pull/12366 should have been labelled "lib / src"
+// https://github.com/nodejs/github-bot/issues/137
+tap.test('label: "lib / src" when 5 or more native files have been changed', (t) => {
```
Comment (addaleax) [inline test/unit/node-labels.test.js]: maybe I’m not understanding the above code, but doesn’t that happen for *4* or more files? :smile:

Comment (phillipj) [inline test/unit/node-labels.test.js]: Good catch, you're absolutely correct! I'll change the related test texts :)

**Comment by mscdex** [comment]
LGTM for now. We can always adjust it later if needed.

**Pull request merged**